package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "puppeteer-utils",
+  "version": "0.0.5",
```

Comment:
Is this intentional?

Reply:
I was having issues with caching in Travis so I had to use this to get it to recognize a new version. I cleared cache now so I have set it to 0.0.1